VERSION

```diff
@@ -1,4 +1,12 @@
 4.14.0+dev0-2021-06-03
 
-# The version string is the first line of this file.
-# It must be in the format described in stdlib/sys.mli
+# Starting with OCaml 4.14, although the version string that appears above is
+# still correct and this file can thus still be used to figure it out,
+# the version itself is actually defined in the build-aux/ocaml_version.m4
+# file (See the OCAML__VERSION* macros there.)
+# To update the present VERSION file:
+# 1. Update build-aux/ocaml_version.m4
+# 2. Run tools/autogen.
+# 3. If you are in a context where version control matters,
+# commit the changes to both build-aux/ocaml_version.m4 and VERSION.
+# The version string must be in the format described in stdlib/sys.mli
```

Comment:
`VERSION` now has more than 9 lines, so `check-typo` expects it to have a valid copyright header. `typo.missing-header` needs to go in `.gitattributes`, therefore.